Remove unicode property names from KuzzleRequest

Aschen opened the pull request:
## What does this PR do ?

In the `KuzzleRequest` object, most of the property were using a special Unicode character to prevents user from modifying them (private attribute emulation).

This was preventing Koncorde to match content of the `KuzzleRequest` object directly.

Those properties now have ASCII names. If an user want to edit one of them he can but Kuzzle may not works after that.

alexandrebouthinon commented on test/api/request/requestInput.test.js:
```diff
@@ -64,46 +64,4 @@ describe('#RequestInput', () => {
-
-  it('should throw if an invalid data parameter is provided', () => {
-    // testing object-only parameters
-    ['volatile', 'body'].forEach(k => {
-      should(function () { new RequestInput({[k]: []}); }).throw(`Attribute ${k} must be of type "object"`);
-      should(function () { new RequestInput({[k]: 123}); }).throw(`Attribute ${k} must be of type "object"`);
-      should(function () { new RequestInput({[k]: false}); }).throw(`Attribute ${k} must be of type "object"`);
-      should(function () { new RequestInput({[k]: 'foobar'}); }).throw(`Attribute ${k} must be of type "object"`);
-    });
-
-    // testing string-only parameters
-    ['controller', 'action', 'jwt'].forEach(k => {
-      should(function () { new RequestInput({[k]: []}); }).throw(`Attribute ${k} must be of type "string"`);
-      should(function () { new RequestInput({[k]: 123}); }).throw(`Attribute ${k} must be of type "string"`);
-      should(function () { new RequestInput({[k]: false}); }).throw(`Attribute ${k} must be of type "string"`);
-      should(function () { new RequestInput({[k]: {}}); }).throw(`Attribute ${k} must be of type "string"`);
-    });
-  });
-
-  it('should not overwrite the controller value if it has already been set', () => {
-    let input = new RequestInput({});
-
-    input.controller = 'foo';
-
-    should(input.controller).eql('foo');
-
-    input.controller = 'bar';
-
-    should(input.controller).eql('foo');
-  });
-
-  it('should not overwrite the action value if it has already been set', () => {
-    let input = new RequestInput({});
-
-    input.action = 'foo';
-
-    should(input.action).eql('foo');
-
-    input.action = 'bar';
-
```
Can this be considered a breaking change since we are changing the KuzzleRequest properties mutation behavior?

xbill82 commented on lib/api/request/requestResponse.ts:
```diff
@@ -154,7 +159,11 @@ export class RequestResponse {
     this.raw = false;
-    this[_request] = request;
+
+    Reflect.defineProperty(this, _request, {
```
Here, `_request` is still `request\u200b`, maybe you wanted to do 
```suggestion
    Reflect.defineProperty(this, 'request', {
```

xbill82 reviewed (approved):
I don't know why you deleted those tests

Aschen commented:
@xbill82 I deleted the test because I also deleted the code asserting correct values for properties.

I wanted to use private property (using `Reflect.defineProperty`) so they don't get printed in the console but I noticed a performance drop:
```
Reflect x 3,194,648 ops/sec ±4.05% (81 runs sampled)
Normal x 60,356,260 ops/sec ±4.02% (83 runs sampled)
Seal x 12,397,200 ops/sec ±3.95% (82 runs sampled)
Fastest is Normal
```

Since this object is crucial and instantiated often I decided to use simple properties without getter/setters returning a private one.  
Of course someone can now mess up with the internal `Request` but since they own the code it's their responsibility
